src/apps/cardano/sign_tx.py

```diff
@@ -184,6 +188,8 @@ def _process_outputs(self):
                     raise wire.ProcessError(
                         "Each output must have address or address_n field!"
                     )
+                if validate_address(output.address) is False:
```

Comment:
why not simply `if not validate_address():` ?

Reply:
fixed